rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/message/RocketChatEditText.java

```diff
@@ -0,0 +1,70 @@
+package chat.rocket.android.widget.message;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.v13.view.inputmethod.EditorInfoCompat;
+import android.support.v13.view.inputmethod.InputConnectionCompat;
+import android.support.v13.view.inputmethod.InputContentInfoCompat;
+import android.util.AttributeSet;
+import android.view.inputmethod.EditorInfo;
+import android.view.inputmethod.InputConnection;
+import android.widget.EditText;
+
+public class RocketChatEditText extends EditText {
```

Comment:
I think the name with feature would be better, such as `ContentComittableEditText` or `ImageKeyboardEditText`.